ops/internal/manage/collect.go

```diff
@@ -92,3 +93,79 @@ func CollectChainConfigs(p string) ([]DiskChainConfig, error) {
 
 	return out, nil
 }
+
+// FindChainConfig searches all superchains for given chain ID
+func FindChainConfig(wd string, chainId uint64) (*DiskChainConfig, config.Superchain, error) {
```

Comment:
This method doesn't seem to be called anywhere

Reply:
I refactored to always use FindChainConfigs instead of this function. Forgot to remove